charts/vertical-pod-autoscaler/CHANGELOG.md

```diff
@@ -14,6 +14,12 @@
 
 ## [UNRELEASED]
 
+## [v1.8.1] - 2025-03-06
+
+### Fixed
+
+- Fixed the name of the secret mounted for TLS certificates when using a custom Issuer with cert-manager. ([#1137](https://github.com/stevehipwell/helm-charts/issues/1137)) _@ahamlinman_
```

Comment:
```suggestion
- Fixed the name of the secret mounted for TLS certificates when using a custom Issuer with cert-manager. ([#1140](https://github.com/stevehipwell/helm-charts/issues/1140)) _@ahamlinman_
```

Reply:
Committed the suggestion, let me know if you'd prefer it squashed.